caesar_cipher/lib/caesar_cipher.rb

```diff
@@ -0,0 +1,33 @@
+class CaesarCipher
+  @@alphabet_length = 'z'.ord - 'a'.ord + 1
```

Comment:
if you've noticed we rarely use these, a constant is better suited as well though